# Message containing relevant times of a task 

string robot_id
string task_type
float64 finishedAt
float64 assignedAt
float64 startedAt
float64 endBatteryLevel
# transportation task only
uint32 request_id
float64 startedPickUpAt
float64 finishedPickUpAt
float64 startedDropOffAt

#charging task only
float64 arrived_at


